content/3_Silver/Intro_Sorted_Sets.mdx

```diff
@@ -197,6 +222,53 @@ System.out.println(set.higher(23));  // ERROR, no such element exists
 
 </JavaSection>
 
+<PySection>
+
+Python does not have a sorted set, however, you can represent a similar data
+structure using a lazy implementation with lists, sets, and binary search. A
+more accurate representation of a sorted set in Python is more complex and not
+silver difficulty (usually involves creating a tree-based data structure).
+
+In the lazy implementation, removing and inserting take $\mathcal{O}(N)$ time
+since a list is used, while checking if an element is present takes
+$\mathcal{O}(1)$ time.
```

Comment:
ain't gon lie this lowk defeats the whole purpose of a sorted set 😭

Reply:
Should I put in an actual sorted set implementation in Python?

In the meantime ill try to find a silver implementation.

Reply:
idk usually when you need a sorted set in python it's just "cry about it and use c++"

Reply:
Ideas
- Show how to create a sorted set in Python (AVL Tree?) (although I would rather not want to code that since I'm not too familiar with making SBBTs) (another option is to link a resource, I found this one to be very detailed and well-made https://nibmehub.com/opac-service/pdf/read/Data%20Structures%20and%20Algorithms%20in%20Python.pdf#page=503)
- The thing about the above is that implementing a sorted set from scratch in a competitive programming context is needlessly time-consuming and probably not helpful.
- Or just state that there is no straightforward, built-in implementation in Python, and to C++ or Java for that instead lol.

Any thoughts?

Reply:
i'm a proponent of 3, though some rare ojs like leetcode do have [sortedcontainers](https://pypi.org/project/sortedcontainers/) build in, which may or may not be worth mentioning

Reply:
i agree with three as well, although it might be useful to link the resource from (1). if you do, you should make it clear that they should read it only if they're curious and that it's not required for silver.

Reply:
Alright, all those sound good. I didn't know Leetcode has sorted containers built in (maybe because it evens the playing field).